worblehat-acceptancetests/src/main/java/de/codecentric/worblehat/acceptancetests/adapter/SeleniumAdapter.java

```diff
@@ -48,8 +48,10 @@ public void initSelenium() {
             LOGGER.error("Error initializing Webdriver", e);
             System.out.println(e);
             System.out.println(e.getMessage());
+            throw e;
         }
 
+        // TODO: don't hardcode linux file separators
```

Comment:
Since Java 7 we can use the `File.separator` constant

Reply:
Didn't know about that. Fixed